rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -1061,17 +997,8 @@ def test_add_blockchain_accounts_with_tags_and_label_and_querying_them(rotkehlch
     )
     assert_proper_response(response)
 
-    # Now add 3 accounts. Some of them  use these tags, some dont
+    # Now add 3 accounts. Some of them use these tags, some doesn't
```

Comment:
Please don't change comments to introduce typos

```suggestion
    # Now add 3 accounts. Some of them use these tags, some don't
```